freshmaker/views.py

```diff
@@ -386,6 +385,24 @@ def _validate_rebuild_request(request):
         if data.get(key) and not isinstance(data[key], str):
             return json_error(400, 'Bad Request', f'"{key}" must be a string.')
 
+    def _event_state_check(freshmaker_event_id):
+        event = models.Event.get_by_event_id(db.session, freshmaker_event_id)
+        event_state = event.state
+        if event_state in (0, 1):
+            return json_error(400, 'Bad Request', f'"{event.id}" is in Initialized or Building state. If you want to continue, use "force": True option.')
+
+    if not data.get("force", None):
+        if 'freshmaker_event_id' in data:
+            _event_state_check(data.get('freshmaker_event_id'))
+        if 'errata_id' in data:
+            events = []  # Todo
+            for event in events:
+                _event_state_check(event)
+        if 'bundle_images' in data:
+            events = []  # Todo
```

Comment:
@qixiang Is there any method or function which would return a list of events by bundle_images?

Reply:
hm, you can ignore the bundle case at this moment, the feature actually doesn't work as expected, we're working on fixing this, sorry for didn't clarify this before.